testsuite/integration-arquillian/tests/other/sssd/src/test/java/org/keycloak/testsuite/sssd/SSSDTest.java

```diff
@@ -10,11 +10,7 @@
 import org.apache.commons.configuration.PropertiesConfiguration;
 import org.jboss.arquillian.graphene.page.Page;
 import org.jboss.logging.Logger;
-import org.junit.Assume;
-import org.junit.Before;
-import org.junit.BeforeClass;
-import org.junit.Rule;
-import org.junit.Test;
+import org.junit.*;
```

Comment:
A small detail: Can you remove star imports (and update your IDE to avoid using star imports and star static imports?)

AFAIK we discussed this in the team a long time ago to avoid star imports (although I think it is not formally documented anywhere)